shared/lib/utils.py

```diff
@@ -125,3 +126,19 @@ def remove_punctuations(s, include_comma=False):
 
 def is_debug_mode() -> bool:
   return os.environ.get('DEBUG', '').lower() == 'true'
+
+
+# Converts a passed in object and escapes all the strings in it.
+def escape_strings(data):
+  if isinstance(data, dict):
+    for k, v in data.items():
+      data[k] = escape_strings(v)
+    return data
+  elif isinstance(data, list):
+    for i, item in enumerate(data):
+      data[i] = escape_strings(item)
+    return data
+  elif isinstance(data, str):
+    return str(escape(data))
+  else:
+    return data
```

Comment:
Is the last `else` an error case?

Reply:
no, if it's a number or some other type, we don't have to do anything about it so just return the original item

Reply:
Add comment?

Reply:
done